### PR TITLE
change title on README

### PR DESCRIPTION
**Pull request by asutosh143**

### CODE REVIEW

**Review by asutosh143 — COMMENTED** [review]
do it

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## Welcome to GitHub Games
+## Welcome to awesome game of Tetris
```
Comment (asutosh143) [inline README.md:1]: add

Comment (asutosh143) [reply README.md:1]: add

Comment (asutosh143) [inline README.md:1]: done

**Review by imasutosh17 — APPROVED** [review]

**Pull request merged**